scalalib/overrides-2.11/scala/jdk/CollectionConventers.scala

```diff
@@ -0,0 +1,5 @@
+package scala.jdk
+
+import scala.collection.convert.{DecorateAsJava, DecorateAsScala}
+
+object CollectionConverters extends DecorateAsJava with DecorateAsScala
```

Comment:
We can't do that. Overrides in the scalalib must stay backward and forward binary compatible with the original ones, because Scala Native libraries are compiled against the original .class files.

We cannot use `scala.jdk` from Scala Native projects, anywhere.